sos/plugins/gdm.py

```diff
@@ -15,7 +15,7 @@ class Gdm(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
 
     plugin_name = 'gdm'
     profiles = ('desktop',)
-    packages = ('gdm',)
+    packages = ('gdm', 'gd3',)
```

Comment:
Isn't the added package name rather `gdm3`?

Reply:
Alright, I need to slow down.
Nice catch!